core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -167,7 +167,7 @@ class RequestSendThread(val controllerId: Int,
 
   override def doWork(): Unit = {
 
-    def backoff(): Unit = CoreUtils.swallowTrace(Thread.sleep(300))
+    def backoff(): Unit = CoreUtils.swallowTrace(Thread.sleep(100))
```

Comment:
300ms seemed a little high. We tend to use 100ms or less as the default in the clients.